CalendarApp/NUnitTestProject/Common/IContainerRegistrationModule.cs

```diff
@@ -0,0 +1,7 @@
+namespace NUnitTestProject.Common
```

Comment:
Is this interface still required for dependency injection configuration purpose?